### PR TITLE
Kaggle benchmark

### PR DESCRIPTION
**Pull request by aptsunny**
*Issue #, if available:*

*Description of changes:*

Major Additions:
- Add kaggle competitions training script
- Update classification task benchmark 
- Add generate_csv_submission function
- Improved Logging, Leaderboard, Shell script, Info
- Bug Fixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhanghang1989** [comment]
@aptsunny This PR generally looks good to me. Could you fix the CI error?

**Comment by aptsunny** [comment]
> @aptsunny This PR generally looks good to me. Could you fix the CI error?

thanks, hang. I fix the bug right now.

**Comment by szha** [comment]
Job PR-179-10 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-179/10/index.html

**Comment by szha** [comment]
Job PR-179-15 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-179/15/index.html

File: autogluon/task/image_classification/pipeline.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 import warnings
 import logging
-
 import mxnet as mx
 from mxnet.gluon import nn
 from mxnet import gluon, init, autograd, nd
 from mxnet.gluon.data.vision import transforms
 from gluoncv.model_zoo import get_model
-
+import gluoncv as gcv
```
Comment (chongruo) [inline autogluon/task/image_classification/pipeline.py]: duplication of gluoncv

File: examples/image_classification/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+|[Plant Seedlings Classification](https://www.kaggle.com/c/plant-seedlings-classification)|0.97607/0.98362/1.0/9%(77/833)|('resnet50_v1', 'resnet50_v1b', 'resnet50_v1c','resnet50_v1d', 'resnet50_v1s')/(1e-4, 1e-3, log=True)/(0.93, 0.95)/(1e-6, 1e-4, log=True)|120/30/96/2|[log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/plant-seedlings-classification/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/plant.sh) |
+|[The ature Conservancy Fisheries Monitoring](https://www.kaggle.com/c/the-nature-conservancy-fisheries-monitoring)|1.01974/0.90176/0.29535/30%(117/388)|('resnet50_v1')/(1e-4, 1e-2, log=True)/(0.85, 1.00)/(1e-6, 1e-4, log=True)|120/30/96/2 | [log](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/log/fisheries_Monitoring/summary.log )/[Shell script](https://raw.githubusercontent.com/aptsunny/AutoGluonWebdata/master/autogluon/shell/fish.sh) |
```
Comment (chongruo) [inline examples/image_classification/README.md]: nature

Comment (zhanghang1989) [reply examples/image_classification/README.md]: @aptsunny could you send a PR to this repo for webdata https://github.com/zhanghang1989/AutoGluonWebdata

Comment (aptsunny) [reply examples/image_classification/README.md]: Some webdata is constantly updated, I submit the other existing files.

**Pull request merged**